.github/workflows/test-clojure.yml

```
@@ -0,0 +1,33 @@
+name: Test Math/Clojure
+
+on:
+  push:
+    branches: ["dev", "504-clj-tests"]
```

Commit this before merging!

```suggestion
    branches: ["dev"]
```